sched/adaptdl_sched/allocator.py

```diff
@@ -102,62 +184,66 @@ async def _find_nodes(self):
         node_template = NodeInfo(max_resources, True)
         return node_infos, node_template
 
+    def _get_job_info(self, job, job_infos, allocations=None):
+        if allocations and job.get("status", {}).get("phase") \
+                not in ["Pending", "Running", "Starting", "Stopping"]:
```

Comment:
This condition doesn't seem to be the same as before. Why is it changed?

Reply:
allocations is None means that I am only parsing one job (called from allocate job), so it should get rid of this phase condition because that phase is async and shouldnt matter.